### ISSUE TITLE
Question About Register Image .tif on adata

### ISSUE THREAD

**Issue by Wooseok1114** [scverse/squidpy#632]
I have adata with raw counts in .X, spatial coordinate in .obsm['spatial'].
I have also tissue image in .tif format 

My question is
1. How can I add this image to adata.uns['spatial'] ?

I have loaded the image using 
`plt.imread("image.tif") `
and it has dimensions of (26452 rows x 26458 columns)

I then attempted to add it to adata.uns['spatial'] using the following code:
```
adata.uns['spatial'] = {}
adata.uns['spatial']['Mouse'] = {}
adata.uns['spatial']['Mouse']['images'] = {}
adata.uns['spatial']['Mouse']['images']['hires'] = image.astype(np.float32)
```
However, when I use 
`sq.pl.spatial_scatter(adata, color="leiden")`
the image only shows as a vertical line. Is there a correct way to attach the image to adata.uns['spatial']?

2. Should I crop the image?

The tissue is only located in the region of the image that spans from [5000:13000, 12000:20000], so the majority of the image is empty. 
Would it be beneficial to crop the image? 

If so, should the cropping be done before adding the image to adata.uns['spatial']? 

Additionally, if the image is cropped, should the adata.obsm['spatial'] field also be adjusted accordingly? 
...

**Comment by giovp** [comment]
hi @Wooseok1114 ,

can you share the shape of the image? It should be (H,W) or (H,W,C). What do you mean by straight line?

**Comment by giovp** [comment]
mmh interesting, this is really weird, what do you use to do the plotting that looks like a straight line?

**Comment by Wooseok1114** [comment]
image = plt.imread("image.tif")

adata = sc.read_h5ad("counts.h5ad") #h5ad file with x, y coordinate data on .obsm['spatial']

adata.uns['spatial'] = {}
adata.uns['spatial']['Mouse'] = {}
adata.uns['spatial']['Mouse']['images'] = {}
adata.uns['spatial']['Mouse']['images']['hires'] = image.astype(np.float32)

sq.pl.spatial_scatter(adata, color="leiden")

**Comment by giovp** [comment]
ok, try with shape=None and also can you please print the shape of `adata.obsm["spatial"]`

**Comment by Wooseok1114** [comment]
![image](https://user-images.githubusercontent.com/122770648/215483547-7ec380c8-6530-4a5e-9041-ca5e0a3e7d1d.png)
adata.obsm['spatial'] is this

![image](https://user-images.githubusercontent.com/122770648/215483862-871fb403-8790-439a-888e-0f0fd0456899.png)
adata.obs have also the x, y coordinate

![image](https://user-images.githubusercontent.com/122770648/215483966-1e57a266-6a0a-4f13-9197-e086a495a739.png)
shape = None brought this result

**Comment by giovp** [comment]
nice, so can you now try to add a dimension to the image before assigning it to adata.uns, e.g.
```python
import numpy as np

image = ... # read image
image = image[..., np.newaxis]
...
```

**Comment by Wooseok1114** [comment]
![image](https://user-images.githubusercontent.com/122770648/215523667-cbb4e651-a0b1-48e5-bad1-fb5185703608.png)
it works!

So my last two questions are
1. I found the tissue image is grayscale (1 channel) image. However, in python, this image was shown in blue to yellow colorscale. How can I set this image in grayscale even after I load it on adata.uns?
2. The image is too big. Almost 70% of image is empty. The spatial_scatter also give very low resolution dots. Is there any way to load image 
![image](https://user-images.githubusercontent.com/122770648/215524748-5e162061-2491-4e35-8135-1754e5fa06c6.png)
, which is matched to this shape = None plot?
